internal/controller/sleepinfo/sleepinfo_controller.go

```diff
@@ -204,7 +205,7 @@ func (r *SleepInfoReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&kubegreenv1alpha1.SleepInfo{}).
 		WithOptions(controller.Options{
-			MaxConcurrentReconciles: 20,
+			MaxConcurrentReconciles: r.MaxConcurrentReconciles,
```

Comment:
If MaxConcurrentReconciles is not set, it would be 0. How it work the NewControllerManagedBy? Should we use our default instead?
From main there is already the default, but here were the controller is configured it's better to use a default maybe if not passed by the parent?

Reply:
Yes, I meant a default also inside the controller not only on the flag. But I see that by default with 0 it is set to 1, so it could be ok.